src/sonic-yang-mgmt/sonic_yang_ext.py

```diff
@@ -874,8 +897,10 @@ def _revXlateContainerInContainer(self, model, yang, config, table):
         if yang.get(modelContainer['@name']):
             config[modelContainer['@name']] = dict()
             self.sysLog(msg="revXlateContainerInContainer {}".format(modelContainer['@name']))
+            self.elementPath.append(modelContainer['@name'])
             self._revXlateContainer(modelContainer, yang[modelContainer['@name']], \
                 config[modelContainer['@name']], table)
+            self.elementPath.pop()
```

Comment:
We need to change these fields in backend eventually. IMHO we should list xpath of these fields in a list\map. And do the special processing when xpath matches. Let me know you view?

Reply:
I am not sure that how xpath could work in such case. IMO, if the current solution is a valid fix, I would prefer not to introduce another solution as this is just a workaround which will be replaced by a final solution(change these fields in backend) in future.

For the xpath you mentioned, could you elaborate how it will work in following functions:
 ```python

    def _findYangTypedValue(self, key, value, leafDict):

        # convert config DB string to yang Type
        def _yangConvert(val):
            # Convert everything to string
            val = str(val)
            # find type of this key from yang leaf
            type = leafDict[key]['type']['@name']

            if 'uint' in type:
                vValue = int(val, 10)
            # TODO: find type of leafref from schema node
            elif 'leafref' in type:
                vValue = val
            #TODO: find type in sonic-head, as of now, all are enumeration
            elif 'stypes:' in type:
                vValue = val
            else:
                vValue = val
            return vValue

        # if it is a leaf-list do it for each element
        if leafDict[key]['__isleafList']:
            vValue = list()
            for v in value:
                vValue.append(_yangConvert(v))
        else:
            vValue = _yangConvert(value)

        return vValue

    def _revFindYangTypedValue(self, key, value, leafDict):

        # convert yang Type to config DB string
        def _revYangConvert(val):
            # config DB has only strings, thank god for that :), wait not yet!!!
            return str(val)

        # if it is a leaf-list do it for each element
        if leafDict[key]['__isleafList']:
            vValue = list()
            for v in value:
                vValue.append(_revYangConvert(v))
        elif leafDict[key]['type']['@name'] == 'boolean':
            vValue = 'true' if value else 'false'
        else:
            vValue = _revYangConvert(value)

        return vValue
 ```

Reply:
Had a call with Praveen, will use a hardcodes list to define those fields which are defined as leaf-list but have string value.